src/pages/billing/List.js

```diff
@@ -23,7 +23,7 @@ import ProjectDetail from 'src/pages/billing/Project'
 import validate from 'validate.js'
 
 
-const ownerRole = 'Owner'
+export const ownerRole = 'Owner'
```

Comment:
Could we do this with the `userRole` as well? Seems nicer to me than duplicating strings in Project.js

Reply:
Absolutely, will do.